thirdeye/thirdeye-pinot/pom.xml

```diff
@@ -361,52 +361,6 @@
 
   <build>
     <plugins>
-      <plugin>
-        <artifactId>maven-shade-plugin</artifactId>
```

Comment:
Per our discussion, we are going to publish shaded JAR and source JAR together. We should only remove this plugin once we have both JAR build.

Reply:
The shading is moved to the `thirdeye-dashboard`.